vespalog/src/vespa/log/bufferedlogger.cpp

```diff
@@ -330,10 +341,10 @@ BackingBuffer::log(const Entry& e) const
     if (e._count > 1) {
         std::ostringstream ost;
         ost << e._message << " (Repeated " << (e._count - 1)
-            << " times since " << (e._timestamp / 1000000) << "."
-            << std::setw(6) << std::setfill('0') << (e._timestamp % 1000000)
+            << " times since " << count_s(e._timestamp.time_since_epoch()) << "."
```

Comment:
Since this is to be read by humans, could use this as an opportunity to emit as a date/time instead of a UNIX timestamp

Reply:
Wait for std::format on gcc